JSDemos/Demos/DataGrid/GroupSummaries/description.md

```diff
@@ -1 +1,17 @@
-This example illustrates the use of&nbsp;group data summaries (calculated totals in&nbsp;each group). To&nbsp;configure a&nbsp;group summary, declare the **groupItems** array within the **summary** object. This demo also illustrates the component&rsquo;s ability to&nbsp;position group summary items. By&nbsp;default, they are displayed in&nbsp;the group row&rsquo;s header, but you can shift them to&nbsp;a&nbsp;specific column (the **alignByColumn** property) or&nbsp;even move them to&nbsp;the group footer (the **showInGroupFooter** property).
\ No newline at end of file
+To configure group summaries, populate the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/).[groupItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/) array with summary configuration objects. Each object should specify a [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) that supplies data for summary calculation and a [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType)&mdash;an aggregate function that should be applied to this data.
+
+### Position a Group Summary
+
+Group summaries are displayed in parentheses after the group caption. You can reposition them as follows:
+
+* Display a summary in a column          
+You can align a summary with the column by which it was calculated (see the Sale Amount and Total Amount columns). To do this, enable the [alignByColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#alignByColumn) property. If you want to align a summary with a column of your choice, specify it in the [showInColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInColumn) property.
+
+* Move a summary to the group footer       
+To position a summary in the group footer, enable the [showInGroupFooter](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInGroupFooter) property. The summary will be aligned with the column by which it was calculated (see the Total Amount column). If you want to use another column, specify it in the [showInColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInColumn) property.
```

Comment:
@Abadzhev 
This paragraph explains how to move one summary to the group footer. Not all summaries. That's why singular fits better here. Can this be applied:
"To display a summary value in the group footer, enable the [showInGroupFooter]() property. The Summary will appear in its corresponding column (see the Total Amount column). If you want to use another column, specify it in the [showInColumn] property."
?

Reply:
So let's enumerate the terms that we use: 

- Summary, Total Summary, Group Summary: this is a generic term, a feature name. You use it throughout the topic in the sense of "summary item", or "summary value". Sometimes it works fine. "You can add a group summary to the grid", "The grid supports group summaries", "The footer displays a total summary" - these are fine as synonyms to summary item, or summary value, or as a feature name. 
- Summary item: an object that defines calculation settings. It can produce one value (total summary), or multiple values (group summary). 
- Summary value: the result of summary calculation (as mentioned above, there can be multiple values per summary item). This is what the grid actually displays - calculation results, values. 

Now let's return to the sentence in question, and start with "group footer". You can't use "the group footer", because this phrase implies that it's one and only footer that we know about. But there are multiple group rows (unless there's only one group). You can say "the grid's footer" - that one in fact is specific - only one instance exists per grid / view.

So we need to use plural for "group footer". Then if we do that, then "group footers display summary values". In other words: one summary item produced multiple values and the grid displays those values in group footers. 

If you say "a summary value", you mean one value, that is a single group footer cell.  

I agree my version is not explicit about values being all produced by a single summary item. (I don't think it's necessary to specify, since we talk about summary item settings...) But we can be specific, if you want: 

"You can make a summary item display its values within group footers. Set the ..."  

I hope that you can find a version that works based on these comments.

Reply:
How about substituting "the group footer" for "the footer of its group" ():
"Move a summary to the footer of its group
To position a summary in the footer of its group, enable the showInGroupFooter property. The summary will be aligned to its corresponding column (see the Total Amount column). If you want to use another column, specify it in the showInColumn property."
?

Reply:
It still leaves the definite article, and the meaning is pretty much the same: I don't see much difference between "the group footer" or "the footer of its group". The latter seems a bit more complicated and confusing. I don't know why you are fighting the plural so much :-) A summary item does display values in multiple group footers. I think it's all technically correct and simple enough... 

Display Summary Values within Group Footers
You can make a summary item display its values within group footers. To do that, set ...